kernel/names.ml

```diff
@@ -906,6 +840,9 @@ module GlobRef = struct
 
 end
 
+type global_reference = GlobRef.t
+[@@ocaml.deprecated "Alias for [GlobRef.t]"]
```

Comment:
Where does this come from?

Reply:
Very good point, this is solving a merge problem with the `GlobRef` patch, I thought I had documented it somewhere but let me see.

Reply:
So the `GlobRef` patch removed an indentifier without the proper deprecation but it was because of a rebase problem, long story... Basically the original patch did the move in one step, however another patch got merged so we have to do the move now in two steps `Globnames.global_reference`-> `Names.global_reference` -> `Names.GlobRef.t`. Short story, this is a fix, and intenteded.

Reply:
Oh indeed, sorry I had missed the commit message explaining this.